scripts/apps/authoring/views/authoring-topbar.html

```diff
@@ -5,7 +5,7 @@
   sd-media-query min-width="880"
   data-test-id="authoring-topbar"
 >
-
+    <toggle-full-width ng-click="hideMonitoring(currentRoute.href === item.href, $event)" ng-href="#{{ :: item.href }}" is-expanded="monitoringHidden"></toggle-full-width>
```

Comment:
are arguments to `hideMonitoring` doing anything? When I logged them, it always appears empty